### PR TITLE
Adds french translation file

### PR DESCRIPTION
**Pull request by alemangui**
Fixes #

## Description of the Change
I have added the french translation file.

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: oauth2_provider/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+msgid "Authorize"
+msgstr "Autoriser"
+
+#: templates/oauth2_provider/authorize.html:17
+msgid "Application requires the following permissions"
+msgstr "L'application nécessite les permissions suivantes"
+
+#: templates/oauth2_provider/authorized-token-delete.html:6
+msgid "Are you sure you want to delete this token?"
+msgstr "Êtes-vous sûr de vouloir supprimer ce jeton ?"
```
Comment (alemangui) [inline oauth2_provider/locale/fr/LC_MESSAGES/django.po:181]: This might seem weird but in French there is a space before `?`

Comment (n2ygk) [reply oauth2_provider/locale/fr/LC_MESSAGES/django.po:181]: [Oui !](https://mashable.com/article/leaving-space-before-end-punctuation-texting#:~:text=French.,have%20a%20space%20preceding%20them.)

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 * #1152 `createapplication` management command enhanced to display an auto-generated secret before it gets hashed.
+* #1163 Adds French translations.
```
Comment (n2ygk) [inline CHANGELOG.md:21]:
```suggestion
### Added
* #1163 Adds French translations.

### Changed
* #1152 `createapplication` management command enhanced to display an auto-generated secret before it gets hashed.
```

**Review by n2ygk — CHANGES_REQUESTED** [review]
Thanks for this. See the suggestion.

File: CHANGELOG.md (hunk 2 above)
Comment (alemangui) [reply CHANGELOG.md]: Thanks! :+1: suggestion committed

**Review by n2ygk — APPROVED** [review]

**Pull request merged**